src/go/k8s/kuttl-v2-ci-test.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestSuite
+startKIND: true
+kindContainers:
+  - localhost/redpanda-operator:dev
+  - docker.redpanda.com/redpandadata/redpanda:v23.2.8
+  - docker.redpanda.com/redpandadata/console:v2.3.2
+testDirs:
+  - ./tests/e2e-v2-ci/tests
+kindConfig: ./kind.yaml
+kindNodeCache: false
+commands:
+  - command: mkdir -p tests/_e2e_artifacts_v2
```

Comment:
I think thous should change as we are using the same path in https://github.com/redpanda-data/redpanda/blob/6082c1bc24f54b5a3ede83a9eaf47d8ccd29472b/src/go/k8s/kuttl-v2-test.yaml#L13

Other wise it would create in the local environment result that are hard to differentiate.